src/lib.rs

```diff
@@ -310,13 +335,17 @@ pub fn apply_filter(bpf_filter: BpfProgramRef) -> Result<()> {
     };
     let bpf_prog_ptr = &bpf_prog as *const sock_fprog;
 
+    // Until https://github.com/rust-lang/libc/issues/3342 is fixed, define locally
```

Comment:
nit: move this to the top of the file. it's best to have constants defined there